src/github/client/app-token-holder.ts

```diff
@@ -1,6 +1,10 @@
 import { AsymmetricAlgorithm, encodeAsymmetric } from "atlassian-jwt";
 import AuthToken, { ONE_MINUTE, TEN_MINUTES } from "./auth-token";
 
+//TODO: Remove Probot dependency to find privateKey
+import * as PrivateKey from "probot/lib/private-key";
```

Comment:
/nit we could just import a function to avoid star imports

Reply:
The package doesnt export it and in the long run we will want to remove this dependency and have our own solution.